portal-web/test/functional/com/liferay/portalweb/tests/Homepage.testcase

```diff
@@ -0,0 +1,37 @@
+definition {
+
+	property testray.main.component.name = "Login";
```

Comment:
@Piyush-Gupta10 in order to have this test run in our CI test batches, we'd also want to set something here like
```property portal.upstream = "true";```